src/languages/en.ts

```diff
@@ -4818,7 +4819,7 @@ const translations = {
             }
         },
         genericError: 'Oops, something went wrong. Please try again.',
-        onBehalfOfMessage: (delegator: string) => `on behalf of ${delegator}`,
+        onBehalfOfMessage: ({delegator}: OnBehalfOfParams) => `on behalf of ${delegator}`,
```

Comment:
I will take a look today: https://github.com/Expensify/App/pull/48594#issuecomment-2377962148

Reply:
Isn't this fixed [here](https://github.com/Expensify/App/pull/49826)?

Reply:
ah, he beat me to it! closing this.